### PR TITLE
New package: mypaint-brushes-1.3.0

### PR DESCRIPTION
**Pull request by kartik-ynwa**
Dependency for GIMP 2.10.0

### CODE REVIEW

File: srcpkgs/mypaint-brushes/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'mypaint-brushes'
+pkgname=mypaint-brushes
+version=1.3.0
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake"
+makedepends="libmypaint-devel"
+short_desc="Brushes used by MyPaint and other software using libmypaint"
+maintainer="Kartik Singh <kartik.ynwa@gmail.com>"
+license="CC0-1.0, GPL-2.0, GPL-3.0"
```
Comment (maxice8) [inline srcpkgs/mypaint-brushes/template]: use SPDX identifiers

Comment (kartik-ynwa) [reply srcpkgs/mypaint-brushes/template]: Should it be "GPL-2.0-only" and "GPL-3.0-only"? Using this as reference: https://spdx.org/licenses/

Comment (maxice8) [reply srcpkgs/mypaint-brushes/template]: They should be either -or-later or -only depending on the header files in the source code.

Comment (maxice8) [reply srcpkgs/mypaint-brushes/template]: on this specific case it is GPL-2.0-or-later and GPL-3.0-only

Comment (kartik-ynwa) [reply srcpkgs/mypaint-brushes/template]: Aight done

**Comment by kartik-ynwa** [comment]
Also it needs libmypaint to build which I have added in another PR. I didn't have the foresight and requested them separately. Do you think I should move `mypaint-brushes` to the PR for `libmypaint`?

**Pull request closed without merging**